src/main/java/io/codelex/loops/practice/Exercise_7.java

```diff
@@ -0,0 +1,37 @@
+package io.codelex.loops.practice;
+
+import java.util.Locale;
+import java.util.Scanner;
+
+public class Exercise_7 {
+
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+
+
+        boolean rollAgain = true;
+        int totalPoints = 0;
+
+        System.out.println("Welcome to Piglet!");
+        while (rollAgain) {
+            int roll = (int) Math.floor(Math.random() * 6) + 1;
+
+            if (roll == 1) {
+                System.out.println("You rolled a " + roll);
```

Comment:
Šī rindiņa varētu būt divas rindas augstāk pirms IF, jo kā redzi ar šo rindņu sākas gan IF, gan else bloks. Tāpēc vienkāršāk to iznest pirms. Otra lieta izvadīšanu, cik punktus beigās dabūja var likt vispār ārā no cikla pēc tam, jo abos gadījumos Tu izvadi un pārtrauc ciklu.

Reply:
Darīts!